src/Aspire.Dashboard/Components/Controls/UserProfile.razor

```diff
@@ -1,7 +1,7 @@
-﻿<AuthorizeView>
-    <Authorized>
-        @if (_showUserProfileMenu)
-        {
+﻿@if (_showUserProfileMenu)
```

Comment:
Only include AuthorizeView on the page if the user profile should be displayed (which means the auth mode needs to be a certain value).

This avoids the need to include various authz services in tests when not needed.